IdokladSdk/Models/IssuedTaxDocument/Patch/IssuedTaxDocumentItemPatchModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using IdokladSdk.Models.Base;
+using IdokladSdk.Validation.Attributes;
+
+namespace IdokladSdk.Models.IssuedTaxDocument.Patch
+{
+    /// <summary>
+    /// IssuedTaxDocumentItemPatchModel.
+    /// </summary>
+    public class IssuedTaxDocumentItemPatchModel : ValidatableModel, IEntityId
+    {
+        /// <summary>
+        /// Gets or sets Id.
+        /// </summary>
+        [RequiredNonDefault]
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets Name.
+        /// </summary>
+        public string Name { get; set; }
```

Comment:
Vím, že to není ani v API, ale asi bych doplnil i tam StringLength atribut podle databáze, jinak to za chvíli budeme pro mobilní aplikaci doplňovat. A to asi i do ostatních POST/PATCH modelů, pokud mají nějakou property typu string (možná je tahle jediná).